bika/lims/profiles/default/metadata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>1.2.1</version>
+  <version>1.2.2</version>
```

Comment:
No need to change the version. We increase the version each time we publish a new release (the last published release is 1.2.0 and the next one to be released is 1.2.1). See https://github.com/senaite/senaite.core/releases

Reply:
:+1: